Non-Hermetic Grimoire.md

```diff
@@ -448,3 +448,127 @@ Olibanum 27
 **R**: Voice, **D**: Perpetual, **T**: Ind  
 This spell lets the sorcerer permanently control a Demon, provided that he penetrates his Magic Resistance. This is a more powerful version of *Chains of the Foul Being* but no sorcerer ever cast it because of the fear of demonic repercussions.    
 *(Base 9, +6 Voice, +12 Perpetual)*
+
+# Naranjs (The Cradle & The Crescent)
```

Comment:
Are these **found** in TC&TC or are these custom and built according to the rules in TC&TC ?

Reply:
I think that they're all customs, following the guidelines in TC&TC, but they lack descriptions, I need to add them.

Reply:
Please do so, and update the PR, then :)